tests/test_unit/test_cdf_tk/test_resource_classes/test_transformations.py

```diff
@@ -0,0 +1,70 @@
+from collections.abc import Iterable
+
+import pytest
+
+from cognite_toolkit._cdf_tk.resource_classes.transformations import TransformationYAML
+from tests.test_unit.utils import find_resources
+
+
+def transformation_destination_cases() -> Iterable:
+    transformation_destination = [
+        {
+            "DataModelSource": {
+                "externalId": "tr_first_transformation",
+                "name": "example:first:transformation",
+                "ignoreNullFields": True,
+                "destination": {
+                    "type": "instances",
+                    "dataModel": {
+                        "externalId": "my_data_model",
+                        "version": "1",
+                        "space": "my_space",
+                        "destinationType": "my_view",
+                    },
+                },
+            }
+        },
+        {
+            "ViewDataSource": {
+                "externalId": "tr_first_transformation",
+                "name": "example:first:transformation",
+                "ignoreNullFields": True,
+                "destination": {
+                    "type": "assets",
```

Comment:
This is wrong and should have been captured by the validation.
```suggestion
                    "type": "nodes",
```